guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,202 @@
+image: /assets/images/guide/payments/units-and-symbols/units-and-symbols-preview.jpg
+
+---
+
+<!--
+
+Editor's notes
+
+This page outlines basic considerations for displaying bitcoin and other currency amounts. Due to the huge
+amount of different notation standards around the world, it focuses on bitcoin.
+
+Illustration sources
+
+Header image
+- https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=1642%3A19055
+
+Content images
+- https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=2318%3A61126
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/units-and-symbols.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols@2x.png"
+   mobile = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile.png"
+   mobileRetina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile@2x.png"
+   alt-text = "The amount 1,027 bitcoin in large letters."
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Units and symbols
+
+Bitcoin, bits, sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. While there are no official standards for how they should be used, a good way to achieve converging user experiences is to formalize what is widely used today.
+
+
+## Current adoption
+
+The most common way bitcoin is expressed is as BTC (bitcoin) or sat (satoshi), with 1 bitcoin being 100 million satoshi. The unicode symbol ₿, formalized in June 2017, is also used to represent BTC (bitcoin), but typeface support is still limited. While not as common, other denominations of BTC such as mBTC (millibitcoins), μBTC (bits), as well as msat (millisatoshi) in the Lightning network are sometimes used. The chart below illustrates how each unit relates to the bitcoin unit.
+
+| Unit         | Symbol       | Bitcoin value     |
+| ------------ | ------------ | ----------------- |
+| bitcoin      | BTC or ₿     | 1                 |
+| millibit     | mBTC         | 0.001             |
+| bit          | μBTC         | 0.000 001         |
+| satoshi      | sat          | 0.000 000 01      |
+| millisatoshi | msat         | 0.000 000 000 01  |
+
+For more information, see the Bitcoin Wiki:
+
+- [Units](https://en.bitcoin.it/wiki/Units)
+- [Satoshi](https://en.bitcoin.it/wiki/Satoshi_(unit))
+
+## Recommended interaction
+
+When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satoshi for Lightning wallets. Due to the challenging nature of scanning amounts with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the application's settings) as well as contextually, whenever the value is primarily displayed.
```

Comment:
I feel like we want to put this under {% include tip/recommendation.html %} to emphasize our recommendation
```suggestion
{% include tip/recommendation.html %}

When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satoshi for Lightning wallets. Due to the challenging nature of scanning amounts with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the application's settings) as well as contextually, whenever the value is primarily displayed.

{% include tip/close.html %}
```

Reply:
Good thinking with the tip component. The next few paragraphs are part of this recommendation, so I don't think it works here. But as I took another look, the following paragraph would work well as a recommendation, as it is a direct consideration for product teams. This is the text I'm referring to:

> Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while bitcoin can be used for savings-focused applications.

Reply:
Oh yeah, that one makes more sense.